verifier/testreport.py

```diff
@@ -679,8 +679,11 @@ def characterize_results_by_options(self, test_list, category):
 
             error_detail = test.get('error_detail', None)
             if error_detail:
-                error_keys = error_detail.keys()  # ['options']
-                self.add_to_results_by_key(label, results, error_detail, test, error_keys)
+                try:
```

Comment:
do we need this change to add a `try / catch` ? If it's being set in the C++ executor code, then maybe not? see if we can revert to no try/catch and have it still work.

Reply:
This may be needed in the case of some incomplete error_detail objects.